www/src/pages/getting-started/why-react-bootstrap.mdx

```diff
@@ -0,0 +1,134 @@
+# Why React-bootstrap?
+React-Bootstrap is a complete re-implementation of the
+Bootstrap components using React. It has **no dependency
+on either `bootstrap.js` or jQuery.** If you have React 
+setup and React-Bootstrap installed, you have everything 
+you need. 
+
+Methods and events using jQuery is done imperatively
+by directly manipulating the DOM. In contrast, React
+uses updates to the state to update the virtual DOM.
+In this way, React-bootstrap provides a more reliable
+solution by incorporating Bootstrap functionality into 
+React's virtual DOM. Below are a few examples of how
+React-Bootstrap components differ from Bootstrap.
+
+## A Simple React Component
+The CSS and details of Bootstrap components are rather
+opinionated and lengthy. React-bootstrap simplifies
+this by condensing the original Bootstrap into React-styled
+components. 
+
+### Bootstrap
+```
+import React from 'react';
+
+class Example extends React.Component {
+  render() {
+    return (
+      <div class="alert alert-warning alert-dismissible fade show" role="alert">
+        <strong>Oh snap! You got an error!</strong> 
+        <p> 
+          Change this and that and try again.
+        </p>
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+    )
+  }
+}
+
+```
+
+### React-Bootstrap
+```
+import React, { Component } from 'react';
+import { Alert } from 'react-bootstrap';
+
+class Example extends React.Component {
+  render() {
+    return (
+      <Alert dismissible variant="danger">
+        <Alert.Heading>Oh snap! You got an error!</Alert.Heading>
+        <p>
+          Change this and that and try again.
+        </p>
+      </Alert>;
+    )
+  }
+}
+```
+
+## Bootstrap with state
+Since React bootstrap is built with React Javascript, state
+can be passed within React-bootstrap components as a prop.
+It also makes it easier to manage the state as updates are
+made using React’s state instead of directly manipulating
+the state of the DOM. This also gives a lot of flexibility 
+when creating more complex components.
+
+### React-bootstrap
+```
+class AlertDismissable extends React.Component {
```

Comment:
This looks like a copy/paste of [this](https://react-bootstrap.github.io/components/alerts/#dismissing) example, maybe we should just link to it?

We would need to update this anyways when #3982 gets merged.

Reply:
Yes - I used that example so readers can see it on the same page instead of two separate tabs. I’ll change it if that makes more sense!

Reply:
Maybe we can import and render a `ReactPlayground` component of the `AlertDismissable` example? That way it will keep the example up-to-date while also save users from navigating to another page. 

I'd look at the [Alerts](https://github.com/react-bootstrap/react-bootstrap/blob/master/www/src/pages/components/alerts.mdx) page for how to import and render such component.

Reply:
@bpas247 I didn't initially render the ReactPlayground component since I wasn't sure how to exclude the live preview, and if including and it being inconsistent with the other code examples would be an issue. I have imported and rendered it. Let me know if it is okay?